AntennaTracker/wscript

```diff
@@ -8,6 +8,7 @@ def build(bld):
         vehicle=vehicle,
         libraries=bld.ap_common_vehicle_libraries() + [
             'PID',
+            'AP_Avoidance',
```

Comment:
Libraries that aren't used in all vehicles should be set only in the vehicles that use them.
This library seems to only be used in Copter now, so it shouldn't be in the other vehicles - no reason to compile a library that isn't going to be used.